### DISCUSSION TITLE
Usability Point in Case: Why Simplification is the Ultimate Sophistication

### DISCUSSION THREAD

**Discussion by probonopd** [helloSystem/hello#215] [Show and tell]
Wrote an article that illustrates the design philosophy in helloSystem:

> You may have heard the phrase “Simplification is the ultimate sophistication”. It is attributed to Leonardo da Vinvi and to Steve Jobs. While many user interface designers agree, most fail to deliver. Here is a prime example: __Formatting a disk on Linux and other UNIX-like systems.__

https://medium.com/@probonopd/usability-point-in-case-why-simplification-is-the-ultimate-sophistication-49d074032cd1

**Comment by kettle-7** [comment]
> Formatting a disk on Linux and other UNIX-like systems.

Interesting you mention that actually because after using Windows since XP, I found it much easier after moving to Ubuntu. Of course it'll be different for other people but I see right clicking the disk and clicking **Format** > **ext4** simple enough.

However, choosing the filesystem is the hard part.

**Reply by kettle-7** [reply]
In addition to this, you mentioned renaming disks. That is one thing that surprises me, many open source desktops can't. There is a function in libparted for it, but gnu is not welcome technologies.

**Reply by probonopd** [reply]
> right clicking the disk and clicking Format > ext4

Does Ubuntu have this? Then it's better than I remembered.

> gnu is not welcome technologies

Where did you read that? It's not on https://github.com/helloSystem/hello/wiki/Welcome-and-unwelcome-technologies...

**Reply by kettle-7** [reply]
> Where did you read that?

Earlier in the renaming disks issue I mentioned libparted having a function for it and you said, from memory, that basically you would prefer a non gnu libparted. I'll try find it.

**Reply by kettle-7** [reply]
> Is there a non-GNU libparted? We are trying to stay away from GNU where possible.

helloSystem/Filer#69

**Reply by probonopd** [reply]
Ah, right, it is GPL. We prefer BSD licensed code wherever possible of course.
